nvflare/fuel/hci/server/file_transfer.py

```diff
@@ -37,6 +32,7 @@
 from nvflare.fuel.hci.conn import Connection
 from nvflare.fuel.hci.reg import CommandModule, CommandModuleSpec, CommandSpec
 from nvflare.fuel.hci.zip_utils import convert_legacy_zip, unzip_all_from_bytes, zip_directory_to_bytes
+from nvflare.private.fed.server.job_validator import JobValidator
```

Comment:
Should we import nvflare.private from nvflare.fuel package?

Reply:
This code is totally out of place. It requires some refactoring of the command module to get this working correctly. There is a ticket for that already.